frontend/public/templates/work_packages/tabs/_add_work_package_relation.html

```diff
@@ -9,6 +9,7 @@
     id="relation_to_id-{{ handler.relationsId }}"
     name="relation[to_id][{{ handler.relationsId }}]"
     ng-model="$parent.relationToAddId"
+    title="{{ I18n.t('js.relation_button_inputs.' + handler.relationsId) }}"
```

Comment:
@0xF013 This is pretty generic functionality. I'd suggest we add one key for field prompt (is this the right terminology?) and interpolate the name of the relation.

```js
// en.js.field_value_enter_prompt: "Enter a value for '%s'"
// de.js.field_value_enter_prompt: "Einen Wert für '%s' eingeben"
{{ I18n.t('field_value_enter_prompt' , { field: I18n.t('js.relation_labels.' + handler.relationsId) }) }}
```